src/main/resources/i18n/messages.properties

```diff
@@ -1003,9 +1003,17 @@ MetadataField.type=Type
 
 MetadataTemplatesList.delete-confirm=Delete this template?
 MetadataTemplatesList.empty=There are no metadata templates associated with this project.
-MetadataTemplatesList.remove=Removed
+
+MetadataTemplatesList.default=Default
+MetadataTemplatesList.set-as-default=Set as Default
+MetadataTemplatesList.cannot-remove-default=Cannot remove a default template. Please select another template as a default first
+MetadataTemplatesList.remove=Remove
 MetadataTemplatesList.download=Download
 
+MetadataTemplatesList.allFields=All Fields
+MetadataTemplatesList.allfields-description=This template displays all the metadata fields on a project. Note that setting this as the default could have an adverse effect on the linelist page performance.
```

Comment:
Can you make the naming more consistent i.e. `allFields-description`

Reply:
Yup seems like I forgot to capitalize the `a`. Fixed in 8e63a642f3f46bf750d3af026924c0835221c272